client-akka/src/main/scala/com/full360/prometheus/http/finatra/AkkaHttpCounter.scala

```diff
@@ -19,10 +19,10 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```

Comment:
Everything related to the Client Akka HTTP is in an incorrect package called `finatra`

Reply:
its fixed, thanks for the warning